ResourceStatusSystem/Client/ResourceManagementClient.py

```diff
@@ -219,7 +219,7 @@ def addOrModifyGGUSTicketsCache( self, gocSite = None, link = None, openTickets
   def selectDowntimeCache( self, downtimeID = None, element = None, name = None,
                            startDate = None, endDate = None, severity = None,
                            description = None, link = None, dateEffective = None,
-                           lastCheckTime = None, gocdbServiceType = None, meta = None ):
+                           lastCheckTime = None, gOCDBServiceType = None, meta = None ):
```

Comment:
I know, it's weird indeed. But it was the only way to fix that bug. 
`gOCDBServiceType` becomes `GOCDBServiceType` to match the column name of the schema which is case sensitive.